librz/il/definitions/bitvector.c

```diff
@@ -554,27 +554,28 @@ RZ_API RZ_OWN RzILBitVector *rz_il_bv_complement_2(RZ_NONNULL RzILBitVector *bv)
  * Result of (x + y) mod 2^length
  * \param x RzILBitVector, Operand
  * \param y RzILBitVector, Operand
```

Comment:
While you are at it, please document that both operands must have the same size.